src/main/resources/config/application-buildagent.yml

```diff
@@ -8,8 +8,8 @@ artemis:
     repo-download-clone-path: ./repos-download
     version-control:
         url: http://localhost:8000  # TODO: is this really needed for build agents?
```

Comment:
can we resolve this TODO here?

Reply:
Removed the TODO. The config is required for BuildJobGitService